src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -384,9 +384,7 @@ trait Definitions extends api.StandardDefinitions {
       (sym.name == name) && (sym.owner == PredefModule.moduleClass)
     )
 
-    // This is a not the usual lazy val to prevent it from showing up as a separate module in JavaUniverseForce.scala
-    def getWrapVarargsArrayModule        = if(isNewCollections) ScalaRunTimeModule else PredefModule
-    def wrapVarargsArrayMethod(tp: Type) = getMemberMethod(getWrapVarargsArrayModule, wrapVarargsArrayMethodName(tp))
+    def wrapVarargsArrayMethod(tp: Type) = getMemberMethod(ScalaRunTimeModule, wrapVarargsArrayMethodName(tp))
```

Comment:
Tiny cleanup remains in https://github.com/scala/scala/blob/93178bbfc5a9ff62c182a59a336d32c240d399f1/src/compiler/scala/tools/nsc/transform/CleanUp.scala#L478-L479

The comment should say `scala.runtime` instead of `<Predef-or-ScalaRunTime>`.

Reply:
Nice find!

Reply:
"clean up in CleanUp"

Reply:
Thanks, cleaned up. (Was that the joke?)